app/javascript/packs/components/MainPage.jsx

```diff
@@ -12,9 +38,42 @@ const MainPage = () => {
               <img src="/logo.png" />
               <Text>Welcom to LetsTalk!</Text>
             </LogoTitle>
-            <LoginInput xl={12} placeholder="Login" />
-            <LoginInput xl={12} placeholder="Password" />
-            <LoginButton>Log in</LoginButton>
+            <Formik
+              initialValues={{ login: '', password: '' }}
+              validate={validateValues}
+              onSubmit={onSubmit}
+            >
+              {({ isSubmitting, values, handleChange, errors }) => (
+                <Form>
+                  <Row>
+                    <Field
+                      className={errors.login ? 'invalid' : 'valid'}
+                      name="login"
+                      label="Email"
+                      error={errors.login}
+                      as={LoginInput}
+                      value={values.login}
+                      onChange={handleChange}
```

Comment:
Main point for using `Field` that you don't need to pass `value`, `onChange` explicitly.